docs/src/introduction/faq.md

```diff
@@ -0,0 +1,69 @@
+# FAQ
+
+## Where can I get help?
+
+For usage questions, please contact us via the
+[Julia Discourse](https://discourse.julialang.org/c/domain/opt). If you're
+running into bugs or have feature requests, please use the [GitHub Issue
+Tracker](https://github.com/jump-dev/Convex.jl/issues).
+
+## How does Convex.jl differ from JuMP?
+
+Convex.jl and JuMP are both modelling languages for mathematical programming
+embedded in Julia, and both interface with solvers via MathOptInterface, so many
+of the same solvers are available in both.
+
+Convex.jl converts problems to a standard conic form. This approach requires
+(and certifies) that the problem is convex and DCP compliant, and guarantees
+global optimality of the resulting solution.
```

Comment:
"guarantees" might be a little strong. For example, the second example of @araujoms's nice blog post https://mateusaraujo.info/2023/02/08/sdps-are-not-cheat-codes/ by can be reproduced in Convex easily:

```julia
julia> using Convex, SCS

julia> X = HermitianSemidefinite(3, 3)
Variable
size: (3, 3)
sign: complex
vexity: affine
id: 176…498

julia> p = minimize(real(X[2,2]), X ⪰ 0, X[3,3] == 0, X[2,2] + 2*X[1,3] == 1)
minimize
└─ real (affine; real)
   └─ index (affine; complex)
      └─ 3×3 complex variable (id: 176…498)
subject to
├─ sdp constraint (affine)
│  └─ reshape (affine; real)
│     └─ * (affine; real)
│        ├─ …
│        └─ …
├─ == constraint (affine)
│  ├─ index (affine; complex)
│  │  └─ 3×3 complex variable (id: 176…498)
│  └─ [0;;]
└─ == constraint (affine)
   ├─ + (affine; complex)
   │  ├─ index (affine; complex)
   │  │  └─ …
   │  └─ * (affine; complex)
   │     ├─ …
   │     └─ …
   └─ [1;;]

status: `solve!` not called yet

julia> solve!(p, SCS.Optimizer; silent_solver=true)

julia> p.optval
0.14584807577454767

julia> p.status
OPTIMAL::TerminationStatusCode = 1
```
where as the blog post says, the correct solution is 1. BTW, Clarabel is also pretty far off:
```julia
julia> solve!(p, Clarabel.Optimizer; silent_solver=true)

julia> p.status
OPTIMAL::TerminationStatusCode = 1

julia> p.optval
0.8844497345252251
```
so it's not just SCS.

Maybe:
```suggestion
(and certifies) that the problem is convex and DCP compliant, and guarantees
global optimality of the resulting solution for many classes of problem.
```
or something like that.